noxfiles/ci_nox.py

```diff
@@ -216,6 +216,52 @@ def minimal_config_startup(session: nox.Session) -> None:
     session.run(*start_command, external=True)
 
 
+#################
+## Performance ##
+#################
+@nox.session()
+def performance_tests(session: nox.Session) -> None:
+    """Compose the various performance checks into a single uber-test."""
+    session.notify("teardown")
+    session.run(*START_APP, external=True, silent=True)
+    samples = 2
+    for i in range(samples):
+        session.log(f"Sample {i} of {samples}")
```

Comment:
:trollface: 

```suggestion
        session.log(f"Sample {i + 1} of {samples}")
```

it does read a bit better in the output, i swear!

Reply:
my computer brain saw "0" and thought "this is fine" lol